### PR TITLE
Add $attributes parameter to Html::img()

### PR DESCRIPTION
**Pull request by alien-art**

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 3.0.1 under development
 
-- no changes in this release.
+- new #137: Add $attributes parameter to Html::img()
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- New #137: Add `$attributes` parameter to `Html::img()` (@alien-art)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## 3.0.1 under development
```
Comment (vjik) [inline CHANGELOG.md:3]:
```suggestion
## 3.1.0 under development
```

File: src/Html.php [hunk 3]
```diff
@@ -531,10 +531,14 @@ public static function mailto(string $content, ?string $mail = null, array $attr
      *
      * @param string|null $url The image URL.
      * @param string|null $alt Alt text.
+     * @param array $attributes The tag attributes in terms of name-value pairs.
      */
-    public static function img(?string $url = null, ?string $alt = ''): Img
+    public static function img(?string $url = null, ?string $alt = '',  array $attributes = []): Img
```
Comment (vjik) [inline src/Html.php:536]: @alien-art Can you add tests?

**Review by vjik — CHANGES_REQUESTED** [review]

File: src/Html.php (hunk 3 above)
Comment (alien-art) [reply src/Html.php:536]: added test

**Pull request merged**